ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml

```diff
@@ -0,0 +1,277 @@
+                StatusSwitchTabButton {
+                    text: qsTr("ENS")
+                    fontPixelSize: 13
+                    enabled: false // TODO
+                }
+            }
+            StackLayout {
+                Layout.fillWidth: true
+                currentIndex: tabBar.currentIndex
+                // Tokens layout definition:
+                ColumnLayout {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                    StatusPickerButton {
+                        Layout.fillWidth: true
+                        Layout.topMargin: 8
+                        Layout.preferredHeight: 36
+                        bgColor: Theme.palette.baseColor5
+                        contentColor: Theme.palette.directColor1
+                        text: root.tokenName
+                        font.pixelSize: 13
+                        image.source: root.tokenImage
+                        onClicked: loader.sourceComponent = tokensExtendedView
+                    }
+
+                    StatusInput {
+                        Layout.fillWidth: true
+                        minimumHeight: 36
+                        maximumHeight: 36
+                        topPadding: 0
+                        bottomPadding: 0
+                        text: root.tokenAmountValue == 0 ? "" : root.tokenAmountValue.toString()
+                        font.pixelSize: 13
+                        rightPadding: amountText.implicitWidth + amountText.anchors.rightMargin + leftPadding
+                        input.placeholderText: "0"
+                        validationMode: StatusInput.ValidationMode.IgnoreInvalidInput
+                        validators: StatusFloatValidator {  bottom: 0 }
+
+                        StatusBaseText {
+                            id: amountText
+                            anchors.right: parent.right
+                            anchors.rightMargin: 13
+                            anchors.verticalCenter: parent.verticalCenter
+                            text: qsTr("Amount")
+                            color: Theme.palette.baseColor1
+                            font.pixelSize: 13
+                        }
+                        onTextChanged: root.tokenAmountValue = Number(text)
+                    }
+
+                    // Just a filler
+                    Item { Layout.fillHeight: true}
+
+                    StatusButton {
+                        enabled: d.ready
+                        text: qsTr("Add")
+                        height: 44
+                        Layout.alignment: Qt.AlignHCenter
+                        Layout.fillWidth: true
+                        onClicked: { root.addToken(root.tokenAmountValue.toString() + " " + root.tokenName, root.tokenImage, root.operator) }
+                    }
+                } // End of Tokens Layout definition
+
+                // TODO
+                Rectangle {
+                    Layout.fillWidth: true
```

Comment:
can we do it an Item instead?

Reply:
Sure, I will update it when I do the rebase!!